include/boost/math/quadrature/gauss.hpp

```diff
@@ -80,7 +81,7 @@ class gauss_detail<T, 7, 0>
 public:
    static std::array<T, 4> const & abscissa()
    {
-      static const std::array<T, 4> data = {
+      static constexpr std::array<T, 4> data = {
```

Comment:
Yup that's a good diff to make.

Reply:
In this situation no; you only need `constexpr const` for reference variables.